gym-service/src/test/java/org/example/integration/steps/auth/SignUpSteps.java

```diff
@@ -0,0 +1,79 @@
+package org.example.integration.steps.auth;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import lombok.RequiredArgsConstructor;
+import org.json.JSONObject;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.transaction.annotation.Transactional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@RequiredArgsConstructor
+@Transactional
+public class SignUpSteps {
+    private final MockMvc mockMvc;
+
+    private MvcResult mvcResult;
+    private String requestBody;
+
+    @Given("the user service is running")
+    public void theUserServiceIsRunning() {
+        assertNotNull(mockMvc);
```

Comment:
this given makes no sense. it's ok to skip given and have test only with when->then if there's nothing to provide here